packages/template-retail-react-app/package.json

```diff
@@ -16,6 +18,9 @@
     "format": "pwa-kit-dev format \"**/*.{js,jsx}\"",
     "lint": "pwa-kit-dev lint \"**/*.{js,jsx}\"",
     "lint:fix": "npm run lint -- --fix",
+    "pkg-version": "node ../../scripts/independent-pkg-version.js",
+    "pwa-kit-deps-version:latest": "node ../../scripts/pwa-kit-deps-version.js latest",
+    "pwa-kit-deps-version:sync": "node ../../scripts/pwa-kit-deps-version.js sync",
```

Comment:
Would these scripts visible on a generated projects? I don't think it makes sense on a generated project

Reply:
Hmm, I see what you mean @alexvuong . I don't mind leaving those scripts in the generated project. But we can also choose to remove them. /cc @bendvc

Reply:
Let's make a decision here so we can merge @bendvc @vmarta

Reply:
I think we should remove these scripts while generating a project because they are not executable on a generated project, and a user may confused and wonder what those scripts do in the context of a generated project

Reply:
Agree with @alexvuong , it's weird to have the versioning scripts in `template-retail-react-app`, can we move these scripts to the root of monorepo?

Reply:
yea I think these scripts should stay at root